packages/amplify-e2e-tests/src/__tests__/layer.test.ts

```diff
@@ -20,4 +20,28 @@ describe('amplify add lambda layer', () => {
     await initJSProjectWithProfile(projRoot, {});
     await addLayer(projRoot);
   });
+
+  it.only('init a project and add/update simple layer', async () => {
```

Comment:
probably remove .only() unless you intend for the other tests to be ignored for now